index.js

```diff
@@ -4,7 +4,29 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    console.log(process.argv)
+    if (process.argv[2] === undefined)
```

Comment:
If i run `node run.js 1 2 3`, I get `Invalid operation: 1`. How can you update this conditional to make sure you get the correct error message?

Also, be careful with your formatting. The code in the if block should be indented once.